### ISSUE TITLE
Have a recently added / News panel in homepage

### ISSUE THREAD

**Issue by pierrelambert** [redis-developer/redis-developer#211]
It would be nice for frequent visitors to have a recently added content panel + a News one regarding releases of Redis Ecosystem.

**Comment by ajeetraina** [comment]
Good idea @pierrelambert It would be great to have featured articles too.
